app/services/queue_manager.rb

```diff
@@ -0,0 +1,53 @@
+class QueueManager
+  attr_reader :page, :job_type
+
+  def QueueManager.push(page, job_type:)
```

Comment:
yeah `class << self` is verbose for one method... this does kind of appeal to me cause you see the method as it would actually be called, but we're doing it the other way in like 20 other places in the codebase so I feel like we should be consistent